spec/models/iso_datastore_spec.rb

```diff
@@ -20,16 +20,7 @@
       context "supports api4" do
         let(:supported_api_versions) { %w(3 4) }
         it "send the method to ovirt services v4" do
-          expect_any_instance_of(ManageIQ::Providers::Redhat::InfraManager::OvirtServices::Strategies::V4)
-            .to receive(:advertised_images)
-          advertised_images
-        end
-      end
-
-      context "does not support api4" do
-        let(:supported_api_versions) { ["3"] }
-        it "send the method to ovirt services v4" do
-          expect_any_instance_of(ManageIQ::Providers::Redhat::InfraManager::OvirtServices::Strategies::V3)
```

Comment:
I'm surprised this isn't showing as conflicting, I removed this in https://github.com/ManageIQ/manageiq/pull/19499/files#diff-74b0b47136c77206a39c6aad71e63599L23-L32

Reply:
Merged cleanly on the command-line